netmiko/fortinet/fortinet_ssh.py

```diff
@@ -39,8 +39,12 @@ def session_preparation(self):
         time.sleep(0.3 * self.global_delay_factor)
         self.clear_buffer()
 
-    def disable_paging(self, delay_factor=1, **kwargs):
-        """Disable paging is only available with specific roles so it may fail."""
+    def disable_paging(
+        self, command: str = None, delay_factor: float = 1, *args, **kwargs
+    ) -> str:
+        """Disable paging is only available with specific roles so it may fail.
+        Does not use `command`, `cmd_verify`, or `pattern` parameters"""
+
```

Comment:
similar concern as above.  since users might be depending on `command` being first positional parameter I thought might be worth special handling here.  IMO it's probably worth a `logger.warning()` or something.

Reply:
`command` would have been broken as the first positional argument (in the old code) for the Fortinet driver...so no one could have actually used command as the first positional argument for that driver.